ubuntu20.04/val-verde-platform-sdk-yarn-yautja-server-cross

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+set -e
+
+# Source package build helper functions
+source ${BUILD_PACKAGE_PREFIX}/bin/${VAL_VERDE_GH_TEAM}-platform-sdk-bash-source-scripts
+
+# Set all environment vars
+export SOURCE_PACKAGE_NAME=yautja-server
+export SOURCE_ROOT=/sources/${SOURCE_PACKAGE_NAME}
+export STAGE_ROOT=/sources/build-staging/${SOURCE_PACKAGE_NAME}-${HOST_OS}${HOST_OS_API_LEVEL}-${HOST_ARCH}
+export QUALIFIED_PACKAGE_NAME=${SOURCE_PACKAGE_NAME}-${HOST_OS}${HOST_OS_API_LEVEL}-${HOST_ARCH}
+export INSTALL_ROOT=${TEMPDIR}/org.${PACKAGE_BASE_NAME}/${QUALIFIED_PACKAGE_NAME}
+
+export PACKAGE_NAME=${PACKAGE_BASE_NAME}-${SOURCE_PACKAGE_NAME}-${HOST_OS}${HOST_OS_API_LEVEL}-${HOST_ARCH}
+export PKG_CONFIG_PATH=${PACKAGE_PREFIX}/lib/pkgconfig:/usr/lib/${BUILD_TRIPLE}/pkgconfig:/usr/share/pkgconfig
+
+# Configure & build the package
+rm -rf ${STAGE_ROOT}
+GIT_URL="https://github.com/${PACKAGE_BASE_NAME}/${SOURCE_PACKAGE_NAME}.git" \
+git-clone val-verde-mainline
+
+mkdir -p ${INSTALL_ROOT}${NODE_PATH}/${SOURCE_PACKAGE_NAME}
+
+rsync -aPx ${SOURCE_ROOT}/ \
+           ${STAGE_ROOT}
+cd ${STAGE_ROOT}
+
+AR=${AR:-${BUILD_PACKAGE_PREFIX}/bin/llvm-ar} \
```

Comment:
The AR and CC defaults can be moved into yarn-build function and applied to the yarn compile step. That way if you want to override AR or CC do so before calling yarn-build.

Reply:
updated